Doc/library/bdb.rst

```diff
@@ -166,45 +214,46 @@ The :mod:`bdb` module also defines two classes:
    Normally derived classes don't override the following methods, but they may
    if they want to redefine the definition of stopping and breakpoints.
 
+   .. method:: is_skipped_line(module_name)
+
+      Return True if *module_name* matches any skip pattern.
+
    .. method:: stop_here(frame)
 
-      This method checks if the *frame* is somewhere below :attr:`botframe` in
-      the call stack.  :attr:`botframe` is the frame in which debugging started.
+      Return True if *frame* is below the starting frame in the stack.
 
    .. method:: break_here(frame)
 
-      This method checks if there is a breakpoint in the filename and line
-      belonging to *frame* or, at least, in the current function.  If the
-      breakpoint is a temporary one, this method deletes it.
+      Return True if there is an effective breakpoint for this line.
+
+      Check for line or function breakpoint and if in effect.  Delete temporary
```

Comment:
I need to defer to @terryjreedy for this suggestion and the ones for the user* functions.  

@vadmium, what we were trying to do was first add docstrings to bdb.py and then retrofit those docstrings back into the online docs.  I may not have applied the changes in the best way when trying to convert to markup, so I'll need his help to figure out how to make sense of it.